pyproject.toml

```diff
@@ -81,9 +81,11 @@ doc = [
     "sphinx_design>=0.5.0",
     "readthedocs-sphinx-search",
     # for unreleased changes
-    "anndata[dev-doc]",
+    "anndata[dev-doc,dask]",
+    "awkward>=2.3",
 ]
 dev-doc = ["towncrier>=24.8.0"] # release notes tool
+test-full = ["anndata[test,lazy]"]
```

Comment:
Maybe a different name?

Reply:
in scanpy we have
- `test-min` which is used in the minimum deps job,
- `test` which is a healthy subset of functionality, and
- `test-full`, which is everything (except for external I think)